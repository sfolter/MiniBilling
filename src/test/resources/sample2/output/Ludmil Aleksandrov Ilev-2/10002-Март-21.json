{
    "documentDate": "2022-08-18T16:36:15.921472900",
    "documentNumber": "10002",
    "consumer": "Ludmil Aleksandrov Ilev",
    "reference": "2",
    "totalAmount": "124.35 EUR",
    "totalAmountWithVat": "149.2200 EUR",
    "lines": [
        {
            "index": 1,
            "quantity": 106.4,
            "lineStart": "2021-01-01T08:00:00Z",
            "lineEnd": "2021-02-28T23:59:59Z",
            "product": "gas",
            "price": 0.4,
            "priceList": 2,
            "amount": "21.76 EUR"
        },
        {
            "index": 2,
            "quantity": 45.6,
            "lineStart": "2021-03-01T00:00:00Z",
            "lineEnd": "2021-03-25T13:40:00Z",
            "product": "gas",
            "price": 4.4,
            "priceList": 2,
            "amount": "102.59 EUR"
        }
    ],
    "taxes": [
        {
            "index": 1,
            "lines": 1,
            "name": "Standing charge",
            "quantity": 58,
            "unit": "days",
            "price": 1.6,
            "amount": "47.45 EUR"
        },
        {
            "index": 2,
            "lines": 2,
            "name": "Standing charge",
            "quantity": 24,
            "unit": "days",
            "price": 1.6,
            "amount": "19.63 EUR"
        }
    ],
    "vat": [
        {
            "index": 1,
            "lines": 1,
            "taxes": 0,
            "taxedAmountPercentage": 60,
            "percentage": 20,
            "amount": "4.3520 EUR"
        },
        {
            "index": 1,
            "lines": 1,
            "taxes": 0,
            "taxedAmountPercentage": 60,
            "percentage": 20,
            "amount": "4.3520 EUR"
        },
        {
            "index": 1,
            "lines": 1,
            "taxes": 1,
            "taxedAmountPercentage": 60,
            "percentage": 20,
            "amount": "4.3520 EUR"
        }
    ]
}