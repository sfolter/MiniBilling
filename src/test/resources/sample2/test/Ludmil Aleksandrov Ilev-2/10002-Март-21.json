{
    "documentDate": "2022-08-16T15:34:31.510644100",
    "documentNumber": "10002",
    "consumer": "Ludmil Aleksandrov Ilev",
    "reference": "2",
    "totalAmount": "126.56USD",
    "totalAmountWithVat": "151.87USD",
    "lines": [
        {
            "index": 1,
            "quantity": 106.4,
            "lineStart": "2021-01-01T08:00:00Z",
            "lineEnd": "2021-02-28T23:59:59Z",
            "product": "gas",
            "price": 0.21,
            "priceList": 2,
            "amount": "22.15USD"
        },
        {
            "index": 2,
            "quantity": 45.6,
            "lineStart": "2021-03-01T00:00:00Z",
            "lineEnd": "2021-03-25T13:40:00Z",
            "product": "gas",
            "price": 2.29,
            "priceList": 2,
            "amount": "104.41USD"
        }
    ],
    "vat": [
        {
            "index": 1,
            "lines": 1,
            "percentage": 20,
            "amount": "4.43USD"
        },
        {
            "index": 2,
            "lines": 2,
            "percentage": 20,
            "amount": "20.88USD"
        }
    ]
}