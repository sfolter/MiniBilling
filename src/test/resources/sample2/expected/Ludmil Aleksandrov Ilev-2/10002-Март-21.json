{
  "documentNumber": "10001",
  "consumer": "Ludmil Aleksandrov Ilev",
  "reference": "2",
  "totalAmount": 273.6,
  "lines": [
    {
      "index": 1,
      "quantity": 367.55,
      "lineStart": "2021-01-01Ð¢14:40:00Z",
      "lineEnd": "2021-02-28T20:59:59Z",
      "product": "gas",
      "price": 0.7,
      "priceList": 1,
      "amount": 257.29
    },
    {
      "index": 2,
      "quantity": 68.45,
      "lineStart": "2021-02-28T20:59:59Z",
      "lineEnd": "2021-03-11T07:00:00Z",
      "product": "gas",
      "price": 1.4,
      "priceList": 1,
      "amount": 95.83
    }
  ]
}